enterprise/server/remote_execution/dirtools/dirtools.go

```diff
@@ -357,6 +359,33 @@ func UploadTree(ctx context.Context, env environment.Env, dirHelper *DirHelper,
 				if err != nil {
 					return nil, err
 				}
+
+				// OutputFileSymlinks and OutputDirectorySymlinks are deprecated in REAPI v2.1,
+				// but Bazel still uses them as of v6.2.0.
+				//
+				// TODO(sluongng): Set OutputSymlinks when Bazel has support for it.
+				// References:
+				// - https://github.com/bazelbuild/remote-apis/blob/35aee1c4a4250d3df846f7ba3e4a4e66cb014ecd/build/bazel/remote/execution/v2/remote_execution.proto#L1071
+				// - https://github.com/bazelbuild/bazel/pull/18198
+				// - https://github.com/bazelbuild/bazel/pull/18202
+				symlinkPath := trimPathPrefix(fqfn, rootDir)
+				ospb := &repb.OutputSymlink{
```

Comment:
uber nit: maybe `symlink` instead of `ospb` which feels a little opaque (what does `pb` stand for?)

Reply:
pb = protobuf... I think I was reusing another variable and never gave it a second thought. 🤣

`symlink` sounds good